test/test.sh

```diff
@@ -183,6 +183,25 @@ test_lp () {
   fi
 }
 
+test_lp_b38 () {
+  local args=()
+  args+=("-log" "${OUTPUT_LOG}")
+  args+=("run")
+  args+=("--assembly" "GRCh38")
+  args+=("--input" "${TEST_RESOURCES_DIR}/lp_b38.vcf.gz")
+  args+=("--output" "${OUTPUT_DIR}")
+  args+=("--GRCh37_annotate_vep_plugin_vkgl" "${TEST_RESOURCES_DIR}/vkgl_public_consensus_empty.tsv")
```

Comment:
bug: `--GRCh37_annotate_vep_plugin_vkgl` -->`--GRCh38_annotate_vep_plugin_vkgl`
This probably explains the positive change in test results, since you are using the public VKGL consensus.